functests/execattachments.py

```diff
@@ -89,7 +89,7 @@ def test_attachment_upload_and_download_allow_not_scanned(self):
 
         with open(self.TEST_IMAGE_DOWNLOAD_PATH, "wb") as fd:
             attachment = self.arch.attachments.download(
-                file_uuid, fd, query={"allow_not_scanned": True}
+                file_uuid, fd, query={"strict": True}
```

Comment:
this now will be denied as the uploaded file will not get scanned before this download attempt

Reply:
good - once the backend PR is in I will actually run these and change to expect failure - and possibly add some more functests